src/modules/windowwalker/app/Window Walker/App.xaml.cs

```diff
@@ -11,5 +12,18 @@ namespace WindowWalker
     /// </summary>
     public partial class App : Application
     {
+        private Mutex _appMutex;
+
+        public App()
+        {
+            // Check if there is already a running instance
+            _appMutex = new Mutex(false, "Global\\" + Guid);

```

Comment:
why `Global\\` though? That could prevent running WindowWalker on a machine with multiple users simultaneously.

Reply:
I don't really know, what should we use?

Reply:
Just removing `Global\\` prefix should be fine.